relay-dynamic-config/src/feature.rs

```diff
@@ -76,15 +76,6 @@ pub enum Feature {
     #[serde(rename = "organizations:continuous-profiling")]
     ContinuousProfiling,
 
-    /// When enabled, every standalone segment span will be duplicated as a transaction.
-    ///
-    /// This allows support of product features that rely on transactions for SDKs that only
-    /// send spans.
-    ///
-    /// Serialized as `projects:extract-transaction-from-segment-span`.
-    #[serde(rename = "projects:extract-transaction-from-segment-span")]
-    ExtractTransactionFromSegmentSpan,
```

Comment:
We used to keep these features as "deprecated". This is not serving too much of a purpose these days, but I still wanted to point this out. If we no longer plan to do this, let's also remove the other deprecated features in a follow-up.

Reply:
Good point, in this case I believe removing the feature flag is OK though, because it was effectively only read in processing relays and removing it does not change the behavior of outdated external relays.